src/test/java/au/org/emii/ncdfgenerator/GenerationIT.java

```diff
@@ -111,11 +111,12 @@ private void doSimpleQuery(String cql) throws Exception {
 
     @Test
     public void anmn_nrs_ctd_profiles_IT() throws Exception {
-
-        // exception handling needs to be improved a lot...
-        InputStream config = getClass().getResourceAsStream("/anmn_nrs_ctd_profiles.xml");
         String cql = "TIME < '2013-6-29T00:40:01Z' ";
-        doTypenameQuery(config, cql);
+        String layerConfigDir = "./src/test/resources/"; // TODO URL url = getClass().getResource("/")  ; url.toString()...
+        String tmpCreationDir = TMPDIR;
+        NcdfGenerator generator = new NcdfGenerator(layerConfigDir, tmpCreationDir);
+        OutputStream os = new FileOutputStream(TMPDIR + "/output.zip");
+        generator.write("anmn_nrs_ctd_profiles", cql, getConn(), os);
```

Comment:
@anguss00 Uggh, it's because the table in the schema is missing a permission for the db account you're connecting under. Can you try again (I've just added the grant).